### ISSUE TITLE
Linked Data in Jupiter

### ISSUE THREAD

**Issue by weiweishi** [ualbertalib/Hydranorth2#167]
### Description
This is a spike to investigate our needs for linked data in our HydraDAMS, and solutions that we can leverage on to meet these requirements. 

### Scope
- [ ] Evaluating possible solutions for the first iteration of linked data implementation requirements
  - [ ] storing metadata values in an LD-friendly way: either as strings + uris, or as uris alone, with a process to resolve uris to strings when needed (e.g. when displaying or indexing the metadata for humans)
  - [ ] importing controlled vocabularies from LD-enabled external sources for use in selecting and validating string/uri values in appropriate metadata fields. This may be implemented in a more LD-based style (accessing the controlled vocabulary as LD triples from a local cache or a remote LD source), or a less LD-based style (use the imported LD data to populate a local conventional database). In both cases the purpose is to provide controlled vocabulary services to the application.

_
> Proposed by metadata team a phased approach to implementation of linked data to enable ongoing assessment of impact and benefit. We recommend initial phases including:
> phase 1: implement URIs for language, license, item type
> phase 2: implement URIs for place names and controlled subjects
> phase 3: implement URIs for person names and freeform subjects
_

The proposed requirements are tied to our user stories and needs of the metadata team to carry out their work. @zschoenb has done some early evaluation to other community solutions like LDF, QA and QA-LDF - Metadata team will continue to monitor and evaluate these community solutions and the features they provided. 
 
### Out of Scope/Future Consideration/Next Phase

- [ ] retrieving objects based on linked data URIs
- [ ] publishing linked data

**Comment by mbarnett** [comment]
I think we should reserve the term "SPIKE" for development spikes -- this sounds more like stakeholder investigation for documenting long-term needs?

Realistically, if we're going to sanely manage the scope of this project and keep it achievable, this isn't something we should be considering spending any development time on before 1.0 is in Production. There are a 1000 well-documented ways to skin the "autocompleting controlled vocabularies" cat, but it's a nice-to-have and not a critical need at this point in the project. 

We need to get the foundation in place before we start working on the nice-to-haves.

edit: I feel like there are a couple other red flags in this ticket. Let's sync up on this issue so that we can avoid falling back into habits we should be trying to avoid with this new work.

**Comment by leahvanderjagt** [comment]
As discussed yesterday, the implementation of linked data is tied to user
stories and the needs of the metadata team to carry out their work. We will
compose a clarified ticket that will more clearly scope the work. Please
also refer to the project principle and plan documents (which I will be
sending out shortly) for further articulation of this work's role within
this project. Weiwei, I believe you had planned to compose a new ticket, so
if you would like to close this I think you should feel free. Leah

___________________________________________________________




*Leah VanderjagtDigital Repository Services Coordinator University of
Albertat. 780.492.3851 *
*leahv@ualberta.ca <leahv@ualberta.ca> *

**Comment by weiweishi** [comment]
@leahvanderjagt @sfarnel @pbinkley I've revised the ticket to reflect our current requirements for Linked Data in the ticket description. Please free feel to edit the ticket as you see fit.
